src/rabbitstressor.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
I would put this file in a `bin/` directory and not in `src/` since it's not really part of the functioning of the webapp. Then you can remove it from `main.js` as well and just make it directly callable via `node bin/stressor.js` or something of the sort.